src/server/src/Server.cpp

```diff
@@ -253,37 +291,41 @@ Server::State Server::get_state()
     return state;
 }
 
-
 void Server::stop_motion()
 {
     drdStop();
 }
 
-
-void Server::set_state(const State& state)
+void Server::set_state(const State &state)
 {
     mutex_.lock();
     state_ = state;
     mutex_.unlock();
 }
 
-
 void Server::stream_robot_state()
 {
-    double position[3];
+    double position[3]; // position px, py, pz
 
     dhdGetPosition(&position[0], &position[1], &position[2]);
 
-    Vector& position_out = port_position_.prepare();
+    Vector &position_out = port_position_.prepare();
     position_out = Vector(3, position);
     port_position_.write();
 
+    double velocity[3]; // velocity vx, vy, vz
+
+    dhdGetLinearVelocity(&velocity[0], &velocity[1], &velocity[2]);
+
+    Vector &velocity_out = port_velocity_.prepare();
+    velocity_out = Vector(3, velocity);
+    port_velocity_.write();
 
-    double force[3];
+    double force[3]; // force fx, fy, fz
```

Comment:
I think we can remove the comment. If you still want to have it, the adopted convention in the code would be

```
/* Force fx, fy, fz */
```

Reply:
honestly, I would keep it. Just for a nice overview and "clarity" even if it sounds redundant. But to me it's nicer when code is self explaining and not relying on guesses (like it would be if user just assume to find px, py, pz, instead of being able to read it in the comment)

Reply:
Ok for keeping. In that case can you please use the adopted convention for the comments? :)

Reply:
Conventioned adopted

Reply:
Ok thank you!